handler/artifact_upload_handler.py

```diff
@@ -0,0 +1,115 @@
+"""
+A tornado application to handle uploading a conda build artifact to a specified
+file location.
+
+"""
+
+import argparse
+import hashlib
+import hmac
+import os
+import subprocess
+
+import tornado.ioloop
+import tornado.httpserver
+import tornado.web
+
+
+class ArtifactUploadHandler(tornado.web.RequestHandler):
+    """Handle writing conda build artifacts to a channel."""
+
+    def initialize(self, write_path, conda_exe, hash_expected):
```

Comment:
@dkillick This is most likely due to my lack of knowledge, but on [line +72](https://github.com/SciTools-incubator/artifact-upload-handler/pull/1/files#diff-bdc8bbb99038f16e80a37a5309b89f87R72) the `kw` dictionary passed to create the `ArtifactUploadHandler` but the `kw` dictionary has the key `token_hash` ... is this not meant to match with `hash_expected` as per your `initialize` signature?

Reply:
It might need to actually – good spot. I'll take a look.